test/protobuf/decoder_test.exs

```diff
@@ -126,7 +126,7 @@ defmodule Protobuf.Decoder.Test do
       use Protobuf, from: Path.expand("../proto/mumble.proto", __DIR__)
     end
 
-    msg = MumbleProto.Authenticate.new(username: "bitwalker")
+    msg = MumbleProto.MumbleProto.Authenticate.new(username: "bitwalker")
```

Comment:
@mmmries I think we should keep the existing behaviour of `inject`, i.e., that it always nests under whatever level `use` is called at. Since you can call `use Protobuf, ...` in a file without wrapping it in a module definition, this would effectively inject modules globally. Thoughts? I just really really dislike the idea of calling `use Protobuf` within a module definition, but defining things in a completely different context, it's completely unintuitive within Elixir to do that.